### PR TITLE
minor changes for color shake

### PR DESCRIPTION
**Pull request by adangert**

### CODE REVIEW

**Comment by adangert** [comment]
When I changed the roll 3 balls to be less bouncy, it kind of just turned into the regular roll a ball, so i removed it here.

**Comment by thaeli** [comment]
This had a merge conflict with #94 so I went ahead and resolved it in my branch. (Because I renamed the modes there, basically)

**Comment by AEFeinstein** [comment]
This was merged as part of #94

File: firmware/user/modes/mode_roll.c [hunk 1]
```diff
@@ -51,7 +51,7 @@
 #define LEN_PENDULUMG sqrt(3.0)
 #define LEN_PENDULUMB sqrt(5.0)
 #define SPRINGCONSTR 5.0
-#define SPRINGCONSTG 20.0
+#define SPRINGCONSTG 2.0
```
Comment (bbkiwi) [inline firmware/user/modes/mode_roll.c:54]: not used for color motion, the pendulum constants are though

**Pull request closed without merging**